http/headers/Supports-Loading-Mode.json

```diff
@@ -0,0 +1,41 @@
+{
+  "http": {
+    "headers": {
+      "Supports-Loading-Mode": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/HTTP/Headers/Supports-Loading-Mode",
+          "spec_url": "https://wicg.github.io/nav-speculation/prerendering.html#supports-loading-mode",
+          "support": {
+            "chrome": {
+              "version_added": "109",
+              "notes": "Initial implementation supports <code>credentialed-prerender</code> only."
```

Comment:
Sounds like `partial_implementation` is needed here?

Reply:
Yup, good call. Added in my next commit